pkg/login/social/connectors/grafana_com_oauth.go

```diff
@@ -64,7 +64,9 @@ func (s *SocialGrafanaCom) Validate(ctx context.Context, settings ssoModels.SSOS
 		return err
 	}
 
-	// add specific validation rules for GrafanaCom
+	if info.AuthUrl != "" || info.TokenUrl != "" || info.TeamsUrl != "" {
```

Comment:
Should we use the validator funcs we defined for the other providers? Similar to `validation.MustBeEmptyValidator(info.AuthUrl, "Auth URL")`.

Reply:
Thanks, done!